coconut/stubs/coconut/convenience.pyi

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+
+#-----------------------------------------------------------------------------------------------------------------------
+# INFO:
+#-----------------------------------------------------------------------------------------------------------------------
+
+
+"""
+Author: Evan Hubinger
+License: Apache 2.0
+Description: Mypy stub file for convenience.py
+"""
+
+
+#-----------------------------------------------------------------------------------------------------------------------
+# IMPORTS:
+#-----------------------------------------------------------------------------------------------------------------------
+
+
+from typing import Any, Callable, Dict, Iterable, Text, Union
+
+from coconut.command.command import Command
+
+
+#-----------------------------------------------------------------------------------------------------------------------
+# COMMAND:
+#-----------------------------------------------------------------------------------------------------------------------
+
+
+CLI = ... # type: Command
+
+
+def cmd(args: Union[Text, bytes, Iterable], interact: bool) -> Command: ...
+
+
+VERSION = ... # type: Dict[Text, Text]
+
+
+def version(which: Text) -> Text: ...
+
+
+#-----------------------------------------------------------------------------------------------------------------------
+# COMPILER:
+#-----------------------------------------------------------------------------------------------------------------------
+
+
+setup = ... # type: Callable
```

Comment:
Can we put a more specific type here? See [here](http://coconut.readthedocs.io/en/develop/DOCS.html#setup) for an explanation of all the valid flags.